examples/AccountManagement/GetAccountHierarchy.php

```diff
@@ -0,0 +1,230 @@
+                $customer = CustomerServiceClient::parseName($customerResourceName)['customer'];
+                print $customer . PHP_EOL;
+                $seedCustomerIds[] = $customer;
+            }
+            print PHP_EOL;
+        }
+
+        foreach ($seedCustomerIds as $seedCustomerId) {
+            $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+            // Creates a query that retrieves all child accounts of the manager specified in search
+            // calls below.
+            $query = 'SELECT customer_client.client_customer, customer_client.level,'
+                . ' customer_client.manager, customer_client.descriptive_name,'
+                . ' customer_client.currency_code, customer_client.time_zone,'
+                . ' customer_client.id' . ' FROM customer_client';
+
+            // Performs a breadth-first search algorithm to build an associative array mapping
+            // managers to their child accounts ($customerIdsToChildAccounts).
+            $unprocessedCustomerIds = [$seedCustomerId];
+            $customerIdsToChildAccounts = [];
+            $rootCustomerClient = null;
+            while (!empty($unprocessedCustomerIds)) {
+                $customerId = array_shift($unprocessedCustomerIds);
+                // Issues a search request by specifying page size.
+                $response = $googleAdsServiceClient->search(
+                    $customerId,
+                    $query,
+                    ['pageSize' => self::PAGE_SIZE]
+                );
+
+                // Iterates over all rows in all pages to get all customer clients under the
+                // specified customer's hierarchy.
+                foreach ($response->iterateAllElements() as $googleAdsRow) {
+                    /** @var GoogleAdsRow $googleAdsRow */
+                    $customerClient = $googleAdsRow->getCustomerClient();
+                    // The customer client that has level of 0 is the specified customer.
+                    if ($customerClient->getLevelUnwrapped() === 0) {
+                        // Store the root customer client, which is the first encountered customer
+                        // client that has level of 0.
+                        if (is_null($rootCustomerClient)) {
+                            $rootCustomerClient = $customerClient;
+                        }
+                        continue;
+                    }
+
+                    // For all level-1 (direct child) accounts that are a manager account, the above
+                    // query will be run against them to create an associative array of managers to
+                    // their child accounts for printing the hierarchy afterwards.
+                    $customerIdsToChildAccounts[$customerId][] = $customerClient;
+                    if ($customerClient->getManagerUnwrapped()) {
+                        // A customer can be managed by multiple managers, so to prevent visiting
```

Comment:
Is this comment really true? I am unable to create such hierarchy through GoogleAds UI.
I have one regular account `SubAccount` and 3 manager accounts: `RootManager, ManagerA, and ManagerB`
If I try to organize them like:
```
RootManager
>ManagerA
>>SubAccount
```
and 
```
ManagerB
>SubAccount
```

Everything is fine, I try to add `SubAccount` to `RootManager` and get error:
```
RootManager
>SubAccount /// ERROR: Invitee is already managed by another manager in your hierarchy
>ManagerA
>>SubAccount
```

If I try to add `ManagerB` to `RootManager`, I get error:
```
RootManager
>ManagerA
>>SubAccount
>ManagerB /// ERROR: The client would be linked more than once in a hierarchy
>>SubAccount
```

Reply:
Now I realize that my account has access to to both `RootManager` and `ManagerB`, so `SubAccount` would show up twice. Nevermind, thank you!